### PR TITLE
Add spring retry for third-party service calls

### PR DESCRIPTION
**Pull request by baixinsui**
fixes https://github.com/eclipse-xpanse/xpanse/issues/1630

### CODE REVIEW

File: runtime/src/main/resources/application.properties [hunk 1]
```diff
@@ -42,4 +42,7 @@ aes.key.file.name=aes_sec
 aes.cipher.algorithm=AES/CBC/ISO10126Padding
 policy.man.endpoint=http://localhost:8090
 otel.exporter.otlp.enabled=false
-spring.activiti.history-level=full
\ No newline at end of file
+spring.activiti.history-level=full
+git.command.timeout.seconds=10
+spring.retry.max-attempts=5
```
Comment (swaroopar) [inline runtime/src/main/resources/application.properties:47]: pls rename this to http.request.retry.max.attempts

File: runtime/src/main/resources/application.properties [hunk 2]
```diff
@@ -42,4 +42,7 @@ aes.key.file.name=aes_sec
...
+spring.retry.delay-millions=1000
```
Comment (swaroopar) [inline runtime/src/main/resources/application.properties:48]: rename this to http.request.retry.delay.milliseconds

**Review by swaroopar — CHANGES_REQUESTED** [review]

**Comment by Alice1319** [comment]
LGTM

**Review by swaroopar — APPROVED** [review]
LGTM

**Pull request merged**